.operations/CONTRIBUTING.md

```diff
@@ -18,10 +18,10 @@ Our content writing guidelines [can be found here](https://github.com/goldbergyo
 
 ### Handling issues and PRs
 
-- Greet the person who contribute out of a good will
-- If a strighforward fix is suggested (e.g., TYPO, linting fix, grammaer improvement, etc) - Approve and merge
-- When a the proposed PR deals with a more opionionated topic - Encourage at least 2 reviewer and let the PR land for a few days to let more folks chime-in
-- Add the contributor to the contributions credit list with this command: `@all-contributors please add @username for content`
+- Great the contributor with kindness
```

Comment:
It's actually important to remind that the contributor is volunteering (WHY should we be kind)

Reply:
Oops, accidentally made a typo for 'Greet' here. 😅

"Greet the contributor with kindness for their hard work and generous contribution?"